src/components/common/Talk/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from 'react-jss';
+import cx from 'classnames';
+
+import Button from '../../forms/Button';
+import styles from './styles';
+
+const Talk = props => {
+  const { theme, sheet, classes, className, talk, ...etc } = props;
+  const { image, title, description, id } = talk;
+  return (
+    <article className={cx(classes.root, className)} {...etc}>
+      <div className={cx(classes.main, 'main-transition')}>
+        <div
+          className={classes.image}
+          style={{ backgroundImage: `url(${image})` }}
+        />
+        <div className={classes.description}>
+          <h4>{title}</h4>
+          <p dangerouslySetInnerHTML={{ __html: description }} />
+        </div>
+      </div>
+      <div className="cta">
+        <Button
+          palette="text"
+          textAlign="left"
+          children="Watch Now"
+          showArrow
+          href="/"
```

Comment:
The URL should come from a property.

Reply:
This URL should be the link to the page details. It would be like:
```
/talks/2018-03-isomorphic-react-without-node/
```
So the idea is to create the talks pages with that format in mind.